docs/20-Sortable-admin-list.md

```diff
@@ -0,0 +1,272 @@
+
+
+```
+    <!-- admin.xml -->
+    <service id="admin.your_entity" class="YourEntityAdminPath">
+        <tag name="sonata.admin" manager_type="orm" label="Your entity admin" show_mosaic_button="false"/> <!-- disable mosaic button -->
+
+        <argument />
+        <argument>YourEntityPath</argument>
+        <argument>YourEntityAdminControllerPath</argument> <!-- specific controller -->
+
+        <call method="setRegistry">  <!-- to get the last position to create new entity with position -->
+            <argument type="service" id="doctrine"/>
+        </call>
+    </service>
+```
+
+
+```
+class YourEntityAdmin extends AbstractAdmin
+{
+    // Sortable list can be only with one page
+    protected $maxPerPage     = 500;
+    protected $perPageOptions = [500];
+
+    protected $datagridValues = [
+        '_page'       => 1,
+        '_sort_order' => 'ASC',
+        '_sort_by'    => 'position',
+    ];
+
+    ...
+    
+    /**
+     * {@inheritdoc}
+     */
+    protected function configureListFields(ListMapper $listMapper)
+    {
+        $listMapper
+            ->addIdentifier('name', TextType::class, [
+                'label'    => 'Nom',
+                'sortable' => false,   // Sorted only by position asc
+            ])
+            ->add('_action', null, [
+                'actions'      => [
+                    'edit'   => [],
+                    'delete' => [],
+                ],
+                'row_align'    => 'center',
+                'header_style' => 'width:20%',
+            ])
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configureRoutes(RouteCollection $collection)
+    {
+        parent::configureRoutes($collection);
+
+        $collection->add('save_positions');  // New route to save positions
+    }
+
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getTemplate($name)
+    {
+        switch ($name) {
+            case 'list': // Templace with css classes to get drag&drop
+                return 'CanalPlusAwakenAdminBundle:CRUD:list_sortable.html.twig';
+            case 'outer_sortable_list_rows_list': // Templace with css classes to get drag&drop
+                return 'CanalPlusAwakenAdminBundle:CRUD:list_outer_rows_sortable_list.html.twig';
+            default:
+                return parent::getTemplate($name);
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function prePersist($object)
+    {
+        parent::prePersist($object);
+
+        // Set position
+        $connection = $this->doctrine->getConnection();
+        $stmt       = $connection->executeQuery(
+            'SELECT MAX(position) as position FROM your_entity_table;',
+        );
+        $result = $stmt->fetch();
+
+        $object->setPosition($result['position'] + 1);
+
+        return [];
+    }
+}
+```
+
+### Admin controller
+
+A new action `savePositionsAction` for the route `save_positions` that will save new positions of all objects
```

Comment:
Same here, maybe we can provide another AdminController and/or a trait to ease out the installation?

Reply:
Ok, I add a trait to add action method to admin controller